clap-utils/src/offline.rs

```diff
@@ -19,6 +19,12 @@ pub const SIGNER_ARG: ArgConstant<'static> = ArgConstant {
     help: "Provide a public-key/signature pair for the transaction",
 };
 
+pub const DUMP_TRANSACTION_MESSAGE: ArgConstant<'static> = ArgConstant {
+    name: "dump_transaction_message",
+    long: "dump-transaction-message",
+    help: "Display the base64 encoded binary transaction message in sign-only mode",
```

Comment:
So you can only dump the message if you're signing? Is it possible to dump a message without signing?

Reply:
Signing is optional in sign-only mode :upside_down_face: 
```
solana --keypair 11111111111111111111111111111111 transfer \
--sign-only --blockhash 11111111111111111111111111111111 \
--dump-transaction-message \
11111111111111111111111111111111 1 

Blockhash: 11111111111111111111111111111111
Transaction Message: AQAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAIAAAwCAAAAAMqaOwAAAAA=
Absent Signers (Pubkey):
 1111111111111111111111111111111
```

Reply:
If it's _only_ signing, and you're _not_ signing, is it really doing anything? 😝

Reply:
`--sign-only` is in its pupal stage and will soon emerge as an adult `--offline` :wink: